packages/money-legos/maker.ts

```diff
@@ -1,70 +1,77 @@
 import erc20 from "./erc20";
 
-import networkIds from "./networks"
+import networkIds from "./networks";
 
 import makerProxyRegistryAbi from "./abi/makerdao/ProxyRegistry.json";
 import dssCdpManagerAbi from "./abi/makerdao/DssCdpManager.json";
 import dssProxyActionsAbi from "./abi/makerdao/DssProxyActions.json";
 
 const ilks = {
-  "BAT-A": {
+  bat_a: {
+    ilk: "BAT-A",
     token: { address: erc20.bat.address },
     join: {
       address: {
         [networkIds.mainnet]: "0x3D0B1912B66114d4096F48A8CEe3A56C231772cA"
-      },
+      }
     }
   },
-  "ETH-A": {
+  eth_a: {
+    ilk: "ETH-A",
     token: {
       address: Object.keys(networkIds).reduce((acc, x) => {
         // @ts-ignore
         acc[x] = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
This is under `maker.ts` but I thought this was `aave` specific? I'm not sure.

Reply:
Thats true we can just leave it as null